apps/nova/src/components/UpcomingLessons/Content.tsx

```diff
@@ -25,8 +25,9 @@ export const Content: React.FC<{
   fetching: boolean;
   error: boolean;
   hasMore: boolean;
+  refetch?: Void;
```

Comment:
I think using the `Void` type, in general, is considered bad or harmful practice. If it's not feasible or not that necessary and tedious to declare/define a type for this specific function. I'd recommend leaving a comment stating what it's supposed to do.

Reply:
You may write, for instance: `// A callback function that invoked tanstack query refetch method`.